src/compiler/compile.js

```diff
@@ -446,7 +446,7 @@ function makeTextNodeLinkFn (tokens, frag) {
           if (token.html) {
             replace(node, parseTemplate(value, true))
           } else {
-            node.data = value
+            node.data = value === undefined ? '' : value
```

Comment:
Let's use [this function](https://github.com/vuejs/vue/blob/dev/src/util/lang.js#L114) for consistency with normal bindings.